### ISSUE TITLE
Hard to navigate

### ISSUE THREAD

**Issue by Tomgnup** [timelic/latex-matrix-generator#1]
It became particularly hard to navigate around the entries when the matrix I want is large. I hope there was an easier way to navigate with cursors. If there is a way, I hope there would be a tutorial for that.

**Comment by timelic** [comment]
Yes, at that time, I made this project only for beauty, not for practicality. I never thought anyone would use it.

I have slightly updated the code and improved the scrolling. I hope it can help you.
